.circleci/config.yml

```diff
@@ -1,7 +1,10 @@
+orbs:
+  codecov: codecov/codecov@4.1.0
+
 #  Xcode version announcments can be found here: https://discuss.circleci.com/c/announcements/
 #  Each post contains a full image manifest, including iOS runtimes, devices, CocoaPods version, etc.
 anchors:
-  - &latest-xcode    "15.2"
+  - &latest-xcode    "15.2.0"
```

Comment:
I will bump this to `15.4.0` (and iOS `17.4`) on Monday when get _allegedly_ get access to `macos.m1.medium.gen1` since `macos.x86.medium.gen2` doesn't support it.

Reply:
There is currently an error trying to use xcode `15.4` with `macos.m1.medium.gen1`..